README.md

```diff
@@ -1 +1,172 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a href=""></a></li>
+  </ul>
+</details>
+
+<details>
+<summary>CSS</summary>
+  <ul>
+    <li><a href=""></a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+ - Gives you the latest stock prices
+ - You can checkout the change in price
+ - You can also check the change percentage in stock prices
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://stock-prices.onrender.com/">View page</a> <br>
+<a href="https://www.loom.com/share/c0de7458a108469c94905ce5502cf2f9">View presentation</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser
+- node.js
+- react.js
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<code>git clone https://github.com/matee-safi/3-Capstone.git</code>
+
+### Install
+
+Install this project with:
+
+<code>npm i</code>
+
+### Usage
+
+To run the project, execute the following command:
+
+<code>npm run start</code>
+
+### Run tests
+
+<code>npm test</code>
+
+### Deployment
+
+You can deploy this project using:
+
+Get a hosting website and give all of the github files
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 Matee Safi
+
+- GitHub: [@github](https://github.com/matee-safi)
+- Twitter: [@twitter](https://twitter.com/matee_safi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/matee-safi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- We'll keep you posted as soon as we've made new plans
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+You can contribute to this project by writing an issue <a href="https://github.com/matee-safi/3-Capstone/issues/new/choose" >here</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please leave a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<a href="https://www.flaticon.com/free-icons/money" title="money icons">Money icons created by Smashicons - Flaticon</a><br>
+<a href="https://www.flaticon.com/free-icons/arrow" title="arrow icons">Arrow icons created by deemakdaksina - Flaticon</a><br>
+<a href="https://www.flaticon.com/free-icons/close" title="close icons">Close icons created by Darius Dan - Flaticon</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Please make sure to give appropriate credit to the author [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for the original design idea as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/235318040-3c3ba4c1-b722-4ddf-9e83-aa268bcbd308.png)